pkg/apis/componentconfig/types.go

```diff
@@ -101,6 +101,8 @@ type KubeProxyConfiguration struct {
 	// metricsBindAddress is the IP address and port for the metrics server to serve on,
 	// defaulting to 127.0.0.1:10249 (set to 0.0.0.0 for all interfaces)
 	MetricsBindAddress string
+	// enableProfiling enables profiling via web interface on /debug/pprof handler.
```

Comment:
It would be nice to indicate the profiling will be served by the metrics server, as it may not be obvious to user.